headless-server-commerce-app/pom.xml

```diff
@@ -10,13 +10,7 @@
   </parent>
 
   <artifactId>headless-server-commerce-app</artifactId>
-  <description>Commerce Headless Server Application
-
-    To start the preview headless-server-app, run:
-    mvn spring-boot:run
-
-    To start the live headless-server-app, run:
-    mvn spring-boot:run -Dspring-boot.run.profiles=local,preview</description>
+  <name>Commerce Headless Server Application</name>
```

Comment:
Why `<name>`? We don't use `<name>` in the poms of the other repositories either. We just use `<artifactId>` and an optional `<description>`.

Reply:
Didn't think about. Fixed.